GoogleForCreators/web-stories-wp
# Video: Show correct flip state of video when in trim mode

## Context


The video trimmer should display videos with the flipping settings, it has in the element on the canvas. And the video frame strip should do the same.

## Summary



## Relevant Technical Choices



For flipping the frames strip am using  `context.setTransform` instead of save and restore which [can be generally slow ](https://stackoverflow.com/a/38079796)

## To-do


n/a

 ## User-facing changes

Before

![before](https://user-images.githubusercontent.com/276316/142685957-ec35819c-e960-453c-b4d8-0f1db285133f.gif)


After
![fixed](https://user-images.githubusercontent.com/276316/142685859-c1728b93-4a8a-4e29-bcf7-32382966aba2.gif)




## Testing Instructions






This PR can be tested by following these steps:

1. Add a video to the canvas
2. Flip it using the flip tools
3. Enter video trim mode
4. Video and the video frame strip should be still flipped as before
5. Verify that trimming video still works 


## Reviews

### Does this PR have a security-related impact?


n/a

### Does this PR change what data or activity we track or use?


n/a

### Does this PR have a legal-related impact?


n/a

## Checklist




---



Fixes #9589
